src/algorithms/a_star_algorithm.py

```diff
@@ -0,0 +1,128 @@
+from dataclasses import dataclass
+from typing import List, Tuple
+
+from lynx.common.vector import Vector
+
+
+class Node:
+    def __init__(self, parent=None, position=None):
+        self.parent = parent
+        self.position = position
+
+        self.g = 0
+        self.h = 0
+        self.f = 0
+

```

Comment:
Aren't there better names, to make it readable

Reply:
Done

Reply:
Done, but not commited @Krystian030